client/src/components/settings/user-management/user-status-indicator.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function UserStatus ({online}) {
+  return (
+    <svg height="10" width="10">
+      <circle cx="5" cy="5" r="5" stroke="none" fill={online ? 'lightgreen' : 'grey'} />
+    </svg>
```

Comment:
Let's use `className` to specify classes for `online` / `offline` styles (`.less` styles in `themes` folder):

```
.cp-user-status.online {
   fill: @color-green;
}

.cp-user-status.offline {
   fill: @color-grey;
}
```